test/cppsuite/test_harness/runtime_monitor.h

```diff
@@ -49,6 +49,18 @@ get_stat(scoped_cursor &cursor, int stat_field, int64_t *valuep)
     cursor->set_key(cursor.get(), stat_field);
     testutil_check(cursor->search(cursor.get()));
     testutil_check(cursor->get_value(cursor.get(), &desc, &pvalue, valuep));
+    testutil_check(cursor->reset(cursor.get()));
+}
+
+/* Consider generating this programmatically in `stat.py`. */
```

Comment:
Can we have a longer comment here explaining why this is like this? i.e. limitations of the config API. Currently the comment is a bit cryptic without further context.

Reply:
1246500

Reply:
Sorry I mean more along the lines of: "The WiredTiger configuration API doesn't provide a function to convert from string statistic names to statistic id, this function provides that mapping. We should consider generating it programmatically in `stat.py` to avoid having to manually add a condition every time we want to observe a new postrun statistic."

Reply:
Oh right. Sorry, that makes sense.

Reply:
3905e0e